packages/js/src/utils/errors.ts

```diff
@@ -1,19 +1,41 @@
 import { NormalizeKeyPaths } from '@silverhand/essentials';
+import { get } from 'lodash';
 
 const logtoErrorCodes = Object.freeze({
-  idToken: {
-    invalidIat: 'Invalid issued at time',
-    invalidToken: 'Invalid token',
+  id_token: {
+    invalid_iat: 'Invalid issued at time',
+    invalid_token: 'Invalid token',
+  },
+  callback_uri_verification: {
+    redirect_uri_mismatched: 'Redirect URI mismatched',
+    error_found: 'Error found',
+    missing_state: 'Missing state',
+    state_mismatched: 'State mismatched',
+    missing_code: 'Missing code',
   },
 });
 
 export type LogtoErrorCode = NormalizeKeyPaths<typeof logtoErrorCodes>;
 
+const getMessageByErrorCode = (errorCode: LogtoErrorCode): string => {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
Better add to todo and link to a linear issue here.

Reply:
Added LOG-1419, PTAL.